plugins/tours/tests/phpunit/unit/template/renderPostmetaBeforeContent.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ *  Tests for render_postmeta_before_content()
+ *
+ * @since      1.0.0
+ * @author     Robert A. Gadon
+ * @package    spiralWebDb\CornerstoneTours\Template\Tests\Unit
+ * @link       http://spiralwebdb.com
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Template;
+
+use Brain\Monkey\Functions;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+use function spiralWebDb\CornerstoneTours\Template\render_postmeta_before_content;
+
+/**
+ * @covers ::\spiralWebDb\CornerstoneTours\Template\render_postmeta_before_content
+ *
+ * @group tours
+ * @group template
+ */
+class Tests_RenderPostmetaBeforeContent extends Test_Case {
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+
+		Functions\expect( 'genesis' )
```

Comment:
Brain Monkey hasn’t been set up yet, as we do that in the `setUp()` method.

Reply:
@hellofromtonya Updated.